website/content/60-advanced/601-emr-integration/6011-trust/60112-adfs/601201-saml2.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Enable SAML2"
+chapter = true
+weight = 601201
+autoNav = true
++++
+
+<center><h3>Enable SAML2</h3></center>
+
+<div style="text-align: justify">
+
+
+   <ol>
+   
+    <li>Login into Windows using the Remote Desktop as follows </li>
+    <li>Go to EC2 console and select the Windows Instance ( Named : <b>LF-ADFS-Windows2019-Server </b>  ) and click on Connect button.<img src="/images/connect-windows.png" title="Connect to Windows Instance" style="margin:15px 0px; border:1px solid black"/></li>
+    <li>Click on <b>Download Remote Desktop File </b> button to download RDP file and close the windows. <img src="/images/downloadrdp.png" title="Connect to Windows Instance" style="margin:15px 0px; border:1px solid black"/> </li>
+    <li>Double click on RDP file you downloaded, and you will get following dialog, Click Continue and enter password <b>W7Bft?M2VbJSnDaeAC!.3cMUP5Lt(NuF</b><img src="/images/windows-continue.png" title="Connect to Windows Instance" style="margin:15px 0px; border:1px solid black"/><img src="/images/windows-enterpassward.png" title="Connect to Windows Instance" style="margin:15px 0px; border:1px solid black"/></li>
+    <li>Once you login into Windows Server, open PowerShell window by clicking on Windows Start and Select PowerShell<img src="/images/windows-poweshell.png" title="Poweshell" style="margin:15px 0px; border:1px solid black"/></li>
+    <li>Execute the following command to upload the AD FS Metadata xml file into S3 bucket (this metadata data xml file contains self-signed certificate public key, ADFS endpoint details etc) make sure to replace <b style="color:red">account-id</b> with AWS AccountId.<pre>Write-S3Object -BucketName lf-workshop-<b style="color:red">account-id</b> -File C:\Users\Administrator\Downloads\WindowsFederationMetadata.xml -Key metadata\WindowsFederationMetadata.xml</pre> <img src="/images/upload-tos3.png" title="Upload metadata into S3" style="margin:15px 0px; border:1px solid black"/> </li>
+    <li>In the previous step we have uploaded AD FS metadata file (WindowsFederationMetadata.xml) into S3 bucket, you can verify the same in S3 Console inside the bucket <b>lf-workshop-account-id</b> under <b>metadata</b> folder. You  need to download this file into your local computer, we need this file to create SAML provider in AWS in the next chapter.</li>
```

Comment:
Did you find a better way to copy/paste in RDP? I didn't and I had to type the entire line. If you find any, you can add that here, it will help the audience to copy the command directly into the RDP window.

Reply:
You can use Notepad to copy and pate inside the Windows to change the account-Id.  changed the instructions to use  copy/paste into Notepad.